trade-module/src/main/java/io/edpn/backend/trade/application/service/RequestStationRequireOdysseyService.java

```diff
@@ -26,10 +26,9 @@ public boolean isApplicable(Station station) {
 
     @Override
     public void request(Station station) {
-        StationDataRequest stationDataRequest = new StationDataRequest();
-        stationDataRequest.setStationName(station.getName());
-        stationDataRequest.setSystemName(station.getSystem().getName());
-
+        StationDataRequest stationDataRequest = new StationDataRequest(
+                null, station.getName(), station.getSystem().getName()
```

Comment:
requesting module is empty, is this set somewhere else?

ohterwise this must be "trade" here